src/core_plugins/kibana/public/management/sections/indices/_edit_sections.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash';
+export default function GetFieldTypes() {
+
+  return function (indexPattern) {
+    const fieldCount = _.countBy(indexPattern.fields, function (field) {
+      return (field.scripted) ? 'scripted' : 'indexed';
+    });
+
+    _.defaults(fieldCount, {
+      indexed: 0,
+      scripted: 0,
+      sourceFilters: 0
+    });
+
+    return [
+      {
+        title: 'fields',
+        index: 'indexedFields',
+        count: fieldCount.indexed
+      },
+      {
+        title: 'scripted fields',
+        index: 'scriptedFields',
+        count: fieldCount.scripted
+      },
+      {
+        title: 'source filters',
+        index: 'sourceFilters',
+        count: fieldCount.sourceFilters
```

Comment:
The count is updated at https://github.com/elastic/kibana/pull/7402/files#diff-042b9dd3484133b320590d4492f96a4eR85